packages/composites/linear-finance/src/tokenAllocationDeriver.ts

```diff
@@ -0,0 +1,46 @@
+import { types } from '@chainlink/token-allocation-adapter'
+import { Requester } from '@chainlink/ea-bootstrap'
+import { Decimal } from 'decimal.js'
+import { Config } from '@chainlink/types'
+import moment from 'moment-timezone'
+
+interface Portfolio {
+  name: string
+  project_id: string
+  symbol: string
+  weight: number
+  unit: number
+  credibility_rating: string
+}
+
+interface ResponseSchema {
+  data: {
+    index_value: {
+      value: number
+      reference_timestamp_utc: string
+    }
+    portfolio_reference_utc: string
+    portfolio: Portfolio[]
+  }
+}
+
+export const deriveAllocations = async (
+  config: Config,
+  path: string,
+): Promise<types.TokenAllocations> => {
+  const now = moment().tz('GMT').format('YYYY-MM-DD[T]hh:mm:ss')
```

Comment:
Not sure if this is correct.  I need to verify this requirement.

Reply:
Yeah be careful with how the timezone is utilized. I've had lots of trouble with timezones in the past...

Reply:
@boxhock what time should we pass to Linear Finance's API?  Currently I'm just setting whatever time it is in GMT to standardize across all NOPs.

Reply:
Let's please confirm with the SA on this